tests/integration/config/plugins/integration-tests/data/tests/tests_save_load.txt

```diff
@@ -43,6 +43,12 @@ test "Loading and Saving"
 		inject "Three Earthly Barges"
 		call "Load First Savegame"
 		call "Depart"
+		apply
+			"test: Depart: 0 zoom_restore_counter" = "test: Depart: zoom_restore_counter"
+			"test: 0 departed 1" = 1
+			"test: year" = year
+			"test: month" = month
+			"test: day" = day
```

Comment:
```suggestion
```

Reply:
I think it doesnt hurt to keep them, if this test was so hard to debug it was partly because of lack of test conditions
That said most of these could be put in an independant function?